azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<p>Let's change the wording to either explain what meaningful way means or just say we'll first clean up the data.</p><p> </p><p>Did you mean dropoff_datetime instead of dropfoff_datetime?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>